modules/Users/UserViewHelper.php

```diff
@@ -757,8 +757,6 @@ protected function setupEmailSettings() {
             $this->ss->assign('MAIL_SMTPPORT',$mail_smtpport);
             $this->ss->assign('MAIL_SMTPSSL',$mail_smtpssl);
         }
```

Comment:
unused $hide_if_can_use_default variable:
Users can not use default outbound anymore but still calculate this value before this part.